src/Utils/transformations.jl

```diff
@@ -1,4 +1,33 @@
-export transform
+using LinearAlgebra
+
+export transform,
+       backtransform
+
+"""
+    backtransform(Rsets::ReachSolution, options::Options)
+
+Undo a coordinate transformation.
+
+### Input
+
+- `Rsets`  -- flowpipe
+- `option` -- problem options containing an `:transformation_matrix` entry
+
+### Output
+
+A new flowpipe where each reach set has been transformed.
+
+### Notes
+
+The transformation is implemented with a lazy `LinearMap`.
+"""
+function backtransform(Rsets, options::Options)
```

Comment:
The type of `Rsets` should be `ReachSolution`, but that type is not available in this module because it is only defined later in the `ReachSets` module.